src/mocks/MorphoImport.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.0.0;
```

Comment:
```suggestion
pragma solidity 0.8.19;
```
[optional]

Reply:
Do you see any advantage to this? My reason for putting `>=0.0.0` so that only `Morpho.sol` specifies the version (if it changes, there's no need to modify `MorphoImport.sol`), but maybe there's a tradeoff I missed.

Reply:
Ah right I get it, it's related to: https://github.com/morpho-org/metamorpho/pull/388#discussion_r1445082587

See the hardhat CI error. We need to specify 0.8.19 as a version used to compile in hardhat config